src/main/java/com/vmware/vipclient/i18n/messages/service/StringService.java

```diff
@@ -33,27 +33,10 @@
     @SuppressWarnings("unchecked")
     @Deprecated
     public String getString(MessagesDTO dto) {
```

Comment:
Please don't do locale fallback in this method because this will change behavior of existing interface.